devtools/maven/pom.xml

```diff
@@ -212,6 +212,18 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.1.1</version>
```

Comment:
Parent POM defines 3.1.0, is there any specific reason why 3.1.1 is needed here?